js/index.js

```diff
@@ -0,0 +1,50 @@
+const buttonAdd = document.querySelector('#add-btn');
+const bookTitle = document.querySelector('#title');
+const bookAuthor = document.querySelector('#author');
+const bookDisplay = document.querySelector('#book-section');
+
+const books = JSON.parse(localStorage.getItem('books')) || [];
+
+const addBook = (title, author) => {
+  books.push({ title, author });
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you add an ID property to make it easier to remove duplicates values.